src/SmartComponents/HybridInventoryTabs/helpers.js

```diff
@@ -87,3 +91,42 @@ export const useActionResolver = (handleModalToggle) =>
     ],
     []
   );
+
+export const useOnLoad = (filters) => {
+  const store = useStore();
+  return useCallback(
+    ({ mergeWithEntities, INVENTORY_ACTION_TYPES, mergeWithDetail }) => {
+      store.replaceReducer(
+        updateReducers({
```

Comment:
^ Can this be tested as it is pointed out here?

Reply:
@bastilian thank you for the review. I thought about this, but could not find a meaningful way of covering with tests. Happy to implement some suggestions.

Reply:
@mkholjuraev You could make the AsyncComponent mock in the ImmutableDevices test call the function provided via the `onLoad`. Than this code would be at least called. You could also add a separate test for the hook and assert that it does correctly "merge" and provides proper arguments to the `mergeWithEntities` function.